wxBookCtrlBase is only defined if wxUSE_BOOKCTRL==1, so don't try to use it
if wxUSE_BOOKCTRL==0
---
 src/common/dlgcmn.cpp |    2 ++
 1 files changed, 2 insertions(+)

Index: b/src/common/dlgcmn.cpp
===================================================================
--- a/src/common/dlgcmn.cpp
+++ b/src/common/dlgcmn.cpp
@@ -529,6 +529,7 @@ bool wxStandardDialogLayoutAdapter::DoLa
 {
     if (dialog->GetSizer())
     {
+#if wxUSE_BOOKCTRL
         wxBookCtrlBase* bookContentWindow = wxDynamicCast(dialog->GetContentWindow(), wxBookCtrlBase);
 
         if (bookContentWindow)
@@ -564,6 +565,7 @@ bool wxStandardDialogLayoutAdapter::DoLa
             FitWithScrolling(dialog, windows);
         }
         else
+#endif // wxUSE_BOOKCTRL
         {
             // If we have an arbitrary dialog, create a scrolling area for the main content, and a button sizer
             // for the main buttons.

 	  	 
